code/mob/living/critter/bee.dm

```diff
@@ -35,6 +35,7 @@
 	health_burn = 25
 	health_burn_vuln = 0.5
 	metabolizes = 0 // for now?
+	void_mindswappable = TRUE // A dream for some people.
```

Comment:
This won't work, the game simply won't let non-admins become bees, it turns you into a wasp instead.